symengine/dict.h

```diff
@@ -119,7 +122,7 @@ int vec_basic_compare(const vec_basic &a, const vec_basic &b);
 int multiset_basic_compare(const multiset_basic &a, const multiset_basic &b);
 //! \return -1, 0, 1 for a < b, a == b, a > b
 int map_uint_mpz_compare(const map_uint_mpz &a, const map_uint_mpz &b);
-
+int map_int_Expr_compare(const map_int_Expr &a, const map_int_Expr &b);
```

Comment:
Because we want signed polynomial degrees. Isuru mentioned this in another [pull request](https://github.com/symengine/symengine/pull/810#discussion-diff-55114696)